02-express-tutorial/data.js

```diff
@@ -1,3 +1,4 @@
+const { products, people } = require("./data");
```

Comment:
This should be in the `app.js` file! This is currently requiring itself, which would likely be throwing an error if the file were required in `app.js`.

The fact that the same variables are defined twice would at least cause an error.